src/data/Nav.js

```diff
@@ -0,0 +1,18 @@
+export const items = [
+  {
+    name: "About Us",
```

Comment:
for our app router, the path is to about so change the link to about instead of aboutus